### ISSUE TITLE
Add HTML meta tags

### ISSUE THREAD

**Issue by zanderle** [williln/williln.github.io#5]
Things like description, keywords, so google will be happy. But also social meta tags (for facebook, twitter, etc) so that when people share the site over social media it looks pretty :)

**Comment by williln** [comment]
Ooh good idea @zanderle! I probably won't have a chance to work on that for a few days, so feel free to submit a PR in the meantime. 🍭

**Comment by williln** [comment]
@zanderle I added what I thought made sense re: description and keywords. For the social meta tags, am I pretty safe to just copy/paste from here? https://moz.com/blog/meta-data-templates-123 I'm not familiar with those!

**Comment by zanderle** [comment]
Yeah, those seem OK. Are you planning on doing that now? I'll have time a bit later, and can double check what you've copy/pasted, if you want :)

**Comment by williln** [comment]
@zanderle I probably won't get back to it for a few days, but I'll ping you when I do! PRs of course welcome in the meantime :)
